artifactory/commands/golang/go.go

```diff
@@ -139,6 +167,41 @@ func (gc *GoCommand) Run() error {
 
 	return err
 }
+func copyGoPackageFiles(destPath, packageName, rtTargetRepo string, rtDetails auth.ServiceDetails) error {
+	var packageFilesPath, version string
+	packageCachePath, err := executersutils.GetPackagePath()
+	if err != nil {
+		return err
+	}
+	packageNameSplitted := strings.Split(packageName, "@")
+	// The Case the user ask for specifc version
```

Comment:
// The Case the user ask for specifc version -->
// The case where the user asks for a specific version